ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -11,6 +11,8 @@ export AZURE_AUTH_LOCATION=$CLUSTER_PROFILE_DIR/osServicePrincipal.json
 export GOOGLE_CLOUD_KEYFILE_JSON=$CLUSTER_PROFILE_DIR/gce.json
 export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 export OVIRT_CONFIG=${SHARED_DIR}/ovirt-config.yaml
+IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
+export IC_API_KEY
```

Comment:
`/var/run/secrets/ci.openshift.io/cluster-profile/ibmcloud-api-key` is not [found](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/20433/rehearse-20433-pull-ci-openshift-kubernetes-release-4.9-k8s-e2e-gcp/1470523642268880896).  I can't find where that secret is being mounted or initialized(I could just be missing it, but from the perspective of CI, that file isn't found).  A similar reference is also being made in https://github.com/openshift/release/pull/20433/files#diff-1a46129e809990d7d8a81a52a2a48bc8258fae24f171dccb14d3bf00301a3b35R149

Reply:
I'll have to look into how to inject this API Key into the deprovision more, and only for IBM Cloud.

Reply:
Okay, added a check to verify we only perform this for `ibmcloud`
```
if [[ "${CLUSTER_TYPE}" == "ibmcloud" ]]; then
  IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
  export IC_API_KEY
fi
```